consensus/service/src/consensus_service.rs

```diff
@@ -622,7 +622,11 @@ impl<E: ConsensusEnclaveProxy, R: RaClient + Send + Sync + 'static> ConsensusSer
                             )
                         })
                         .ok();
-                    blocks_behind = Some(std::cmp::min(peer_block_height - b, 0));
+                    blocks_behind = if peer_block_height > b {
```

Comment:
Good catch!

You can probably make this a one-liner with `peer_block_height.saturating_sub(b)`, but what you've got here might be more readable. Up to you.

Reply:
Oh neat - I like that! Will switch to `saturating_sub`